events/events-team/operations/presentation-session-room-proctor.md

```diff
@@ -0,0 +1,19 @@
+# Contributor Summit Presentation Sessions Moderator Instructions
+
+Thanks for volunteering to moderate one of the presentation sessions! 
+Presentation sessions are 30 minute sessions in which a speaker is making a slide presentation to the contributors.  
+Your contribution will make sure everyone can share knowledge and collaborate.
+
+To be a successful moderator, you’ll complete the following tasks:
+
+* Arrive at least 5 minutes before the session is scheduled to start
+* Make sure the speaker shows up -- hopefully a few minutes early to setup 
+* Verify AV is working: sound, projection, recording (if applicable), check if there is a second microphone for Q&A. Help troubleshoot any problems with AV, facilities, or event staff.
+* It's best if you sit in the front row
+* Introduce the speaker and session
+* Timekeeping is valuable to make sure the day's schedule goes as planned. Make sure the session both begins and ends on time. Provide a time check with a visual cue at 5 minutes before the session is scheduled to end. At the latest, ensure you end 3 minutes before the start of the next session block.
+* If there is time for Q&A, facilatate the Q&A.
```

Comment:
```suggestion
* If there is time for Q&A, facilitate the Q&A.  Take the handheld mic, if available, and bring it to folks who have questions.  Make sure that several people get to ask their questions, rather than allowing one person to dominate Q&A. Exception: sometimes Q&A will become a discussion between the presenter and relevant SIG leads; you should allow this to proceed.  
```

Reply:
I added the suggestion